stock-quote/src/main/java/com/whz/stockquote/StockQuotePublisher.java

```diff
@@ -12,27 +20,59 @@
 import yahoofinance.Stock;
 import yahoofinance.YahooFinance;
 
+/**
+ * This class can be used to publish stock quote data to RabbitMQ. The stock quotes are obtained
+ * from the "Finance Quotes API for Yahoo Finance" by Stijn Strickx.
+ *
+ * @see <a href="https://github.com/sstrickx/yahoofinance-api">yahoofinance-api by Stijn Strickx</a>
+ * @since 1.0.0
+ */
 public class StockQuotePublisher {
 
-  private static final Logger LOGGER = Logger.getLogger(StockQuotePublisher.class.getName());
+  private static final Logger logger = Logger.getLogger(StockQuotePublisher.class.getName());
```

Comment:
Perhaps you could reconsider whether it makes more sense to [use Slf4j instead of java.util](https://stackoverflow.com/questions/11359187/why-not-use-java-util-logging).